plugin.rb

```diff
@@ -25,101 +19,126 @@ class Engine < ::Rails::Engine
     end
   end
 
-  class Presence::Presence
-    class << self
+  module ::Presence::PresenceManager
+    def self.get_redis_key(type, id)
+      return "presence:#{type}:#{id}"
```

Comment:
There's no need for a `return` here. Ruby always return the value of the last expression